e2e/cypress/integration/organizations.spec.js

```diff
@@ -25,6 +25,10 @@ describe('Organization', () => {
     
     it('Edit Organization Description', () => {
         cy.visit('/organization/edit/cypress-test-org')
+
+        // Hide flask debug toolbar
+        cy.get('#flHideToolBarButton').click();
```

Comment:
Should this be a recurring thing?  Why is this showing up all of a sudden?  Or if it was always there, why is it causing an issue now?

Reply:
Now the dev environment is actually being used properly, so it's a thing now. We've handled this on catalog from the beginning...

Reply:
Got it.  Makes sense (******************nods head******************)